test/fares/fares_test.exs

```diff
@@ -4,7 +4,7 @@ defmodule FaresTest do
 
   import Mox
 
-  alias Dotcom.TripPlan.{NamedPosition, Leg, PersonalDetail, PersonalDetail, TransitDetail}
+  alias Dotcom.TripPlan.{NamedPosition, Leg, PersonalDetail, TransitDetail}
```

Comment:
Can you alphabetize these? Makes it easier to see if there are duplicates.

Reply:
[Done](https://github.com/mbta/dotcom/pull/2257/commits/0bcdf4b52b5d09e99b9b7a58d8a385506239d4ad)